### PR TITLE
L10N-2020-01-31-16821 - translations batch

### PR DESCRIPTION
**Pull request by alexandermccate**

### CODE REVIEW

File: packages/L10n/src/locales/de.yml [hunk 1]
```diff
@@ -79,6 +77,13 @@ de:
       less: Weniger anzeigen
       more_about: Mehr über {{topic}} anzeigen
       less_about: Weniger über {{topic}} anzeigen
+    listBox:
+      trigger:
+        placeholder: Auswählen ...
```
Comment (cristinajimenez) [inline packages/L10n/src/locales/de.yml]: Space not needed before periods

File: packages/L10n/src/locales/fr.yml [hunk 2]
```diff
@@ -79,6 +77,13 @@ fr:
       less: Afficher moins
       more_about: Afficher plus sur {{topic}}
       less_about: Afficher moins sur {{topic}}
+    listBox:
+      trigger:
+        placeholder: Selectionner...
```
Comment (cristinajimenez) [inline packages/L10n/src/locales/fr.yml]: Sélectionner
(missing accent)

File: packages/L10n/src/locales/fr.yml [hunk 3]
```diff
@@ -79,6 +77,13 @@ fr:
...
+      filter:
+        a11y_text: Filtrer
+        no_results_message: Vos résultats ne recherche ne correspondaient à aucune option.
```
Comment (cristinajimenez) [inline packages/L10n/src/locales/fr.yml]: I guess translator meant "Vos résultats de recherche"

**Comment by alexandermccate** [comment]
fixes applied @agherreras @cristinajimenez

**Pull request merged**